Python/BST_implementation.py

```diff
@@ -0,0 +1,131 @@
+class Node:  
```

Comment:
Code formating improper, class functions are not intended properly, too many extra spaces, fix these issues